src/accordion/_base.scss

```diff
@@ -25,8 +30,11 @@
 
 .accordion-section-toggle {
   cursor: pointer;
+  padding: ms(1) 0;
+  margin-bottom: 0;
 
   [data-icon="arrow"] {
+    float: right;
```

Comment:
At the moment is the standard 255 character limit set by default, I sent @ckeisato the longest and shortest title example